configurations/broadcast.template.ini

```diff
@@ -6,11 +6,14 @@ domain = @PRIMARY_MEDIA_SERVER_HOST@
 queryParams = {p}.{e}.{i}
 application.default = kLive
 application.LIVE_STREAM_ONTEXTDATA_CAPTIONS=kLiveOTD
+rtsp_domain = @PRIMARY_RTSP_MEDIA_SERVER_HOST@
```

Comment:
Change PRIMARY_RTSP_MEDIA_SERVER_HOST to PRIMARY_MEDIA_SERVER_HOST.
And consult with Jess regarding adding 2 new template vars "rtsp_port" and the missing "port" she will need to add support for replacing them during install.